### ISSUE TITLE
Documentation for Livy REST API

### ISSUE THREAD

**Issue by nixent** [microsoft/dbt-fabricspark#18]
Is there any documentation for the Livy endpoint, something similar to one for [Synapse](https://learn.microsoft.com/en-us/rest/api/synapse/data-plane/spark-session?view=rest-synapse-data-plane-2020-12-01)?

[Fabric REST APi](https://learn.microsoft.com/en-us/rest/api/fabric/articles/) describes other elements of Fabric but not Livy.

**Comment by jcvdodson** [comment]
There's not yet a full public documentation set for the Livy endpoint, but much of the functionality is documented [here](https://learn.microsoft.com/en-us/rest/api/synapse/data-plane/spark-session?view=rest-synapse-data-plane-2020-12-01).

**Comment by jcvdodson** [comment]
You are correct that a Livy session belongs to a particular workspace. Livy itself doesn't really manage access to tables -- it just executes the SQL statements. A Livy session is considered "hosted" by a specific Lakehouse within the workspace, but as you can see in the [`submitLivyCode`](https://github.com/microsoft/dbt-fabricspark/blob/9373300c925f33873a55321d07dac5c812e65643/dbt/adapters/fabricspark/livysession.py#L275) method, we don't actually submit statements against that Lakehouse, just to the Livy session.

**Comment by nixent** [comment]
Thank you for the clarification.
In case I multiple lakehouses which one you'd recommend setting in the dbt profile?

**Comment by jcvdodson** [comment]
I think they would all be equivalent. To be transparent, though, I haven't tested on large lakehouses. My inclination is to say that the profiles.yml lakehouse should be the largest lakehouse or the one you expect to interact with most, but I'm not sure that this really has any practical effect.

**Comment by nixent** [comment]
Could you provide clarification on why the Livy endpoint is integrated within the lakehouse, whereas other endpoints are located under the workspace? Are there specific design aspects or considerations that users should be aware of in this context?

**Comment by jcvdodson** [comment]
The hosting artifact (the Lakehouse) is there for internal reasons related to authentication flows. In practical effect (i.e. statement execution), Livy is tied to the workspace. The hosting artifact has no impact on statement execution -- so, there aren't any special considerations you need to be aware of when using Livy across Lakehouses within a Workspace.

**Comment by prdpsvs** [comment]
Livy api's are documented in mslearn now. Please see them.
